src/view/com/util/numeric/format.ts

```diff
@@ -1,8 +1,13 @@
-export const formatCount = (num: number) =>
-  Intl.NumberFormat('en-US', {
+import {useLanguagePrefs} from '#/state/preferences'
+import {sanitizeAppLanguageSetting} from '#/locale/helpers'
+
+export const FormatCount = (num: number) => {
```

Comment:
Why was this function capitalized? Usually that is reserved for React Components.

Reply:
I resolve the linting error. It reverted to lowercase.